src/Header.tsx

```diff
@@ -30,6 +32,23 @@ export type AppTab = "visual" | "panel" | "debug";
 
 const Header = () => {
   const { classes } = useStyles();
+  const navigate = useNavigate();
+
+  const ctrlsShortCut = () => {
+    navigate("/controls");
+  };
+
+  const debugShortCut = () => {
+    navigate("/debug");
+  };
+
+  const mainShortCut = () => {
+    navigate("/");
+  };
+
+  useKeyboardShortcut("shift", "C", () => ctrlsShortCut());
```

Comment:
You can inline this by doing `() => {navigate("/debug")}`

Reply:
Done!